docs/installing.rst

```diff
@@ -102,15 +102,10 @@ On Windows [5]_:
 Python and OS Compatibility
 ---------------------------
 
-pip works with CPython versions 2.6, 2.7, 3.2, 3.3, 3.4, 3.5 and also pypy.
+pip works with CPython versions 2.6, 2.7, 3.3, 3.4, 3.5 and also pypy.
 
 pip works on Unix/Linux, OS X, and Windows.
 
-.. note::
-
-  Python 2.5 was supported through v1.3.1, and Python 2.4 was supported through
-  v1.1.
-
```

Comment:
@dstufft your comment came in while mine was being typed. That's a fair point. Maybe we just modify it to be a list of "support for Python X.Y was dropped in version Z"?

But agreed it's a bit of a pain. Happy for you to simply drop it and if I get the time I can trawl the changelogs and add back in a simpler summary.